backend/public-graph/graph/resolver.go

```diff
@@ -356,17 +356,21 @@ func (r *Resolver) GetErrorAppVersion(ctx context.Context, errorObj *model.Error
 	var session *model.Session
 	if err := r.DB.WithContext(ctx).Model(&session).
 		Where("id = ?", errorObj.SessionID).
-		Pluck("app_version", &session).Error; err != nil {
-		if !e.Is(err, gorm.ErrRecordNotFound) {
-			return nil
-		}
+		Pluck("app_version", &session).Error; err == nil && session.AppVersion != nil {
+		return session.AppVersion
+	}
+
+	// guess version from error service
+	if errorObj.ServiceName != "" && errorObj.ServiceVersion != "" {
+		return pointy.String(fmt.Sprintf("%s-%s", errorObj.ServiceName, errorObj.ServiceVersion))
```

Comment:
Curious why we need `ServiceName` here? Wouldn't `ServiceVersion` be the parallel to `session.AppVersion` (i.e. githash, although it can be any user inputted string)? 

Not blocking, assuming this may be hard to update for backwards compatibility, but was a little unsure how this was used.

Reply:
a very good point... i just added this recently and no one is using this versioning for backend errors, so i'll switch this to just `ServiceVersion`